server/python/plugin_remote.py

```diff
@@ -799,6 +799,8 @@ def read_requirements(filename: str) -> str:
                 print(str_requirements)
 
             sys.path.insert(0, zipPath)
+            if not debug:
```

Comment:
why is the debug check necessary?

Reply:
should maybe also remove the zipPath altogether if using unzippedPath?

Reply:
There is a comment higher up in the function that says the debugger wanted a stable path, so zipPath points to the zip file. I am not sure what the technical reasons are for that, so I kept the behavior unmodified for the debug route

Reply:
I think that requirement is no longer necessary as I think the vscode debugger now supports relative paths when resolving remote/local roots. I'll play with this and see.

Reply:
Got it, I'll push up a version that removes it